jobs/roadster.js

```diff
@@ -4,8 +4,7 @@ const shell = require('shelljs');
 const { logger } = require('../middleware/logger');
 
 const SPACEX_API = 'https://stage.spacexdata.com/v4';
-const KEY = process.env.SPACEX_KEY;
-const HEALTHCHECK = process.env.ROADSTER_HEALTHCHECK;
+const { KEY, HEALTHCHECK } = process.env.KEY;
```

Comment:
Not sure why this was changed, since `KEY` and `HEALTHCHECK` are distinctly different values.

Reply:
Yes, sorry my mistake(will fix it).